modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -270,10 +270,23 @@ const EditorManager: EditorManager = {
   },
 
   /**
-   * Overrides the default options for editor instances.
+   * Overrides the default options for editor instances. <code>overrideDefaults</code> replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
+   * <br /><br />
+   * When using the cloud, some of these defaults are required for the cloud-based editor to function.
```

Comment:
Just copying across a fix from what @brian-forte said in the linked PR.
```suggestion
   * When using the Tiny Cloud, some of these defaults are required for the cloud-based editor to function.
```

Reply:
My recollection is that there is also an asciidoc attribute for `Tiny Cloud`.

Yep: `cloudname` (source: https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/antora.yml#L27)

Reply:
You cannot use that in Moxiedoc though, as it's not AsciiDoc based nor is that variable available to it.

Reply:
Oh. Ignore me. I thought this was asciidoc :)